### DISCUSSION TITLE
Multi-segment language prediction

### DISCUSSION THREAD

**Discussion by ngcheeyuan** [openai/whisper#2257] [Q&A]
Has anyone tested this? There is an option for this. Just wanted to see if there's anyone who empirically tested this.

**Comment by glangford** [comment]
> There is an option for this

Can you be more specific? What option are you referring to?  Are you using open source Whisper (this repo) or something else?

**Reply by ngcheeyuan** [reply]
@glangford my apologies your suspicion is right. It is only available on the open source version im working on. Closing discussion.
